src/client/Microsoft.Identity.Client/AuthScheme/IAuthenticationOperation.cs

```diff
@@ -9,12 +9,28 @@ namespace Microsoft.Identity.Client.AuthScheme
     /// <summary>
     /// Used to modify the experience depending on the type of token asked. 
     /// </summary>
-    internal interface IAuthenticationScheme
+    public interface IAuthenticationOperation
     {
         /// <summary>
-        /// Value to log to telemetry to indicate pop usage.
+        /// Value to log to telemetry
+        /// Values available:
+        /// 
+        /// Bearer token type.
+        /// Bearer = 1
+        /// 
+        /// Pop token type.
+        /// Pop = 2,
+        ///
+        /// Ssh-cert token type.
+        /// SshCert = 3,
+        ///
+        /// External token type.
+        /// External = 4,
+        ///
+        /// Cdt token type.
+        /// Cdt = 5
```

Comment:
how do we extend the telemetry?

Reply:
This is a good question. Maybe we should use strings instead of numbers?

Reply:
Yeah, i agree with the string approach. this will allow us to still identify the token extension. We need to know the operation because we need to specify a histogram config for the data to be properly represented in the telemetry pipeline.